dut/drivers.wake

```diff
@@ -87,16 +85,20 @@ def makeBasicDrivers basicDriverPlan =
 
 # External wake target to create basic drivers
 # deviceName => the name of the DUT plan
-global def createBasicDriver dutPlanName =
-  getDevicePlan dutPlanName
-  | omap makeBasicDrivers
+global def createBasicDrivers dts =
```

Comment:
dts or compatibleList?

Reply:
compatibleList I guess? though I don't see why we need this at all since this function isn't used anywhere